client/lib/wpcom-xhr-wrapper/index.js

```diff
@@ -27,7 +27,8 @@ export async function jetpack_site_xhr_wrapper( params, callback ) {
 			'X-WP-Nonce': config( 'nonce' ),
 		},
 		isRestAPI: false,
-		apiNamespace: 'jetpack/v4/stats-app',
+		apiNamespace:
+			params.apiNamespace === 'jetpack/v4' ? params.apiNamespace : 'jetpack/v4/stats-app',
```

Comment:
If the API is `jetpack/v4` then we don't need to prefix it with `stats-app`, which are only for compensated APIs. This allows Odyssey to access `jetpack/v4` APIs, but will probably need nonce etc to function in some cases.

Reply:
The trick to call an existing `jetpack/v4` API is to set `apiNamespace` to `jetpack/v4`